x-pack/plugins/alerting/public/pages/maintenance_windows/translations.ts

```diff
@@ -211,15 +211,15 @@ export const CREATE_FORM_CATEGORY_STACK_RULES = i18n.translate(
 export const CREATE_FORM_SCOPED_QUERY_TITLE = i18n.translate(
   'xpack.alerting.maintenanceWindows.createForm.scopedQuery.title',
   {
-    defaultMessage: 'Define scope of rules',
+    defaultMessage: 'Scope of rules',
   }
 );
 
 export const CREATE_FORM_SCOPED_QUERY_DESCRIPTION = i18n.translate(
   'xpack.alerting.maintenanceWindows.createForm.scopedQuery.description',
   {
     defaultMessage:
-      'Use KQL to further narrow down the alerts affected by this maintenance window.',
+      'Use KQL to filter the alerts affected by the maintenance window. Only one category can be selected when filters are enabled.',
```

Comment:
IMO the fact that it's KQL syntax is less important than why you'd click this, so I'd lead with that. For example:

- Add filters that refine the scope of the maintenance window. You can select only one category when filters are enabled.
- Refine the scope of the maintenance window with KQL filters. If filters are enabled, you can choose only one maintenance window category.
- Limit/refine/filter the alerts affected by the maintenance window. Only one category is permitted when filters are enabled.

I think the first option here is my preference, but happy to discuss further.

```